src/sentry/replays/endpoints/project_replay_summarize_breadcrumbs.py

```diff
@@ -78,8 +77,18 @@ def make_seer_request(request_data: str) -> bytes:
             **sign_with_seer_secret(request_data.encode()),
         },
     )
+
     if response.status_code != 200:
-        raise ParseError("A summary could not be produced at this time.")
+        sentry_sdk.logger.warning(
+            "Replay: Failed to produce a summary for a replay breadcrumbs request",
+            extra={
+                "status_code": response.status_code,
+                "response": response.text,
+                "content": response.content,
+            },
+        )
+
+    response.raise_for_status()
```

Comment:
idk I see the other seer endpoints use this instead -- maybe we're getting a <400 status code?